extensions/positron-assistant/src/participants.ts

```diff
@@ -807,12 +809,9 @@ export class PositronAssistantEditorParticipant extends PositronAssistantPartici
 }
 
 /** The participant used in notebook inline chats. */
-class PositronAssistantNotebookParticipant extends PositronAssistantParticipant implements IPositronAssistantParticipant {
+class PositronAssistantNotebookParticipant extends PositronAssistantEditorParticipant implements IPositronAssistantParticipant {
 	id = ParticipantID.Notebook;
-
-	protected override async getSystemPrompt(request: vscode.ChatRequest): Promise<string> {
-		return await fs.promises.readFile(path.join(MARKDOWN_DIR, 'prompts', 'chat', 'default.md'), 'utf8');
-	}
+	// For now, the Notebook Participant inherits everything from the Editor Participant.
```

Comment:
Thoughts on this approach? Technically the Notebook is still an "editor", so I think we can leave the prompts as-is? I can also create separate equivalent prompts to the ones used by the EditorParticipant if the distinction is important.